roles/precheck/tasks/main.yml

```
@@ -154,26 +168,33 @@
           (inventory_hostname not in _hosts)
           or
           ((_repos is defined) and (_repo in _repos))
-    msg: Please use one of the supported linux distro families to run Ceph
+    msg: Please use one of the supported linux distro families to run Ceph {{ ceph_stable_release }} version
          ({{ ansible_distribution }}/{{ ansible_distribution_major_version }}/{{ _repo }} combination is unsupported).
   vars:
     _repo: "{{ ceph.repo | d('distro') }}"
-    _repos: "{{ _supported[ansible_distribution][ansible_distribution_major_version] }}"
+    _repos: "{{ _supported[ceph_stable_release][ansible_distribution][ansible_distribution_major_version] }}"
     _supported:
-      Debian:
-        '10': []
-        '11': [community]
-        '12': []
-      Ubuntu:
-        '20': [community]
-        '22': [distro, community]
-        '24': []
-      AlmaLinux:
-        '8': [distro, community]
-        '9': [community]
-      RedHat:
-        '8': [community]
-        '9': [community]
+      reef: # Ceph 18.2.z
+        Debian:
+          '11': [community]
+          '12': [community]
+        Ubuntu:
+          '22': [distro, community]
+        AlmaLinux:
+          '9': [] # Not supported: https://tracker.ceph.com/issues/68802
+        RedHat:
+          '9': [] # Not supported: https://tracker.ceph.com/issues/68802
+      squid: # Ceph 19.2.z
+        Debian:
+          '12': [community]
+        Ubuntu:
+          '20': [community]
+          '22': [distro, community]
+          '24': [distro, community]
```

```suggestion
          '24': [distro]
```
I believe we have to disable u24 + community because https://download.ceph.com/debian-squid/dists/ does not provide noble. :shrug:

---

Maybe we could also change `ceph_stable_release` to `ceph.release` ? :thinking: